noxfiles/docker_nox.py

```diff
@@ -132,6 +160,139 @@ def build(session: nox.Session, image: str, machine_type: str = "") -> None:
     session.run(*build_command, external=True)
 
 
+def push_prod(session: nox.Session) -> None:
+    """
+    Contains the logic for pushing the suite of 'prod' images.
+
+    Pushed Image Examples:
+      - ethyca/fides:2.0.0
+      - ethyca/fides:latest
+
+      - ethyca/fides-privacy-center:2.0.0
+      - ethyca/fides-privacy-center:latest
+
+      - ethyca/fides-sample-app:2.0.0
+      - ethyca/fides-sample-app:latest
+    """
+    fides_image_name = get_current_image()
+    privacy_center_image_name = f"{PRIVACY_CENTER_IMAGE}:{get_current_tag()}"
+    sample_app_image_name = f"{SAMPLE_APP_IMAGE}:{get_current_tag()}"
+
+    privacy_center_latest = f"{PRIVACY_CENTER_IMAGE}:latest"
+    sample_app_latest = f"{SAMPLE_APP_IMAGE}:latest"
+
+    fides_buildx_command = generate_multiplatform_buildx_command(
+        [fides_image_name, IMAGE_LATEST], docker_build_target="prod"
+    )
+    session.run(*fides_buildx_command, external=True)
+
+    privacy_center_buildx_command = generate_multiplatform_buildx_command(
+        [privacy_center_image_name, privacy_center_latest],
+        docker_build_target="prod_pc",
+    )
+    session.run(*privacy_center_buildx_command, external=True)
+
+    sample_app_buildx_command = generate_multiplatform_buildx_command(
+        [sample_app_image_name, sample_app_latest],
+        docker_build_target="prod",
+        dockerfile_path="clients/sample-app",
+    )
+    session.run(*sample_app_buildx_command, external=True)
+
+
+def push_dev(session: nox.Session) -> None:
+    """
+    Push the bleeding-edge `dev` images.
+
+    Pushed Image Examples:
+      - ethyca/fides:dev
+      - ethyca/fides-privacy-center:dev
+      - ethyca/fides-sample-app:dev
+    """
+    privacy_center_dev = f"{PRIVACY_CENTER_IMAGE}:dev"
+    sample_app_dev = f"{SAMPLE_APP_IMAGE}:dev"
+
+    fides_buildx_command = generate_multiplatform_buildx_command(
+        [IMAGE_DEV], docker_build_target="prod"
+    )
+    session.run(*fides_buildx_command, external=True)
+
+    privacy_center_buildx_command = generate_multiplatform_buildx_command(
+        [privacy_center_dev],
+        docker_build_target="prod_pc",
+    )
+    session.run(*privacy_center_buildx_command, external=True)
+
+    sample_app_buildx_command = generate_multiplatform_buildx_command(
+        [sample_app_dev],
+        docker_build_target="prod",
+        dockerfile_path="clients/sample-app",
+    )
+    session.run(*sample_app_buildx_command, external=True)
+
+
+def push_git_tag(session: nox.Session) -> None:
+    """
+    Push the an image for whatever tag our commit currently has.
```

Comment:
Should this be reworded?

```suggestion
    Push an image for whatever tag our commit currently has.
```